python-sdk/tutorials/automl-with-azureml/image-classification-multiclass-batch-scoring/auto-ml-image-classification-multiclass-batch-scoring.ipynb

```diff
@@ -538,7 +544,9 @@
     "### Create the parallel-run configuration to wrap the inference script\n",
     "Create the pipeline run configuration specifying the script, environment configuration, and parameters. Specify the compute target you already attached to your workspace as the target of execution of the script. This will set the run configuration of the ParallelRunStep we will define next.\n",
     "\n",
-    "Refer this [site](https://github.com/Azure/MachineLearningNotebooks/tree/master/how-to-use-azureml/machine-learning-pipelines/parallel-run) for more details on ParallelRunStep of Azure Machine Learning Pipelines."
+    "Refer this [site](https://github.com/Azure/MachineLearningNotebooks/tree/master/how-to-use-azureml/machine-learning-pipelines/parallel-run) for more details on ParallelRunStep of Azure Machine Learning Pipelines.\n",
+    "\n",
+    "`run_invocation_timeout` is 60 seconds by default, try to increase it for the cases where you want to generate **explanations** and or use large `mini_batch_size`"
```

Comment:
I'm sorry for this and future grammar comments. I'm nit-picking only because this is public-facing ofc. 
is 60 seconds by default, --> is 60 seconds by default.
and or --> and/or